### DISCUSSION TITLE
The OpenAI api issue

### DISCUSSION THREAD

**Discussion by jeffreylm** [open-webui/open-webui#1097] [General]
Good day Everyone,

I saw the following in the console:
GET /openai/api/models HTTP/1.1" 401 Unauthorized

Is there something I need to do to correct this?

Kind regards,
Jeffrey

**Comment by justinh-rahb** [comment]
Normal and expected if you haven't setup OpenAI (or compatible endpoint) with a key. The 401 unauthorized is being sent from the backend of Open WebUI, the request is not forwarded externally if no key is set.

**Comment by jeffreylm** [comment]
Then I have two questions.

1. What are the benefits of setting up the openAI?
2. How do you set this up?

I am new to OpenWebUi and Ollama. Been using it since last week.

Kind regards and thank you in advance

**Reply by justinh-rahb** [reply]
Unless you have a paid account with OpenAI you can't access their API anyhow. You can ignore this function and continue happily using Ollama's API to power Open WebUI.

**Comment by jeffreylm** [comment]
Ok, clear.

This is a side question. LLava keeps giving me the exact same response no matter what image I give it. Eeven afetr rebooting my system. It is a exact word for word response, so a clone. It has done it 4 times behind each other no matter the image I ask it to define. This happened on the 13b.

Do you have any idea of what is happening?

**Reply by justinh-rahb** [reply]
No, if there's no errors in the WebUI or browser logs it would most likely be a model or an Ollama issue: https://github.com/ollama/ollama

If you still suspect the problem is in WebUI, it would be best to open a new issue for it with logs/screenshots and a sample of the image involved.

**Reply by jeffreylm** [reply]
There is no error in the webUI in the docker logs. It would go off and do it's thing and come back with an exact clone of the previous response of the previous. And it comes from me simply asking it to: Describe this image

That would make it create a clone response of the previous.
